src/Libraries/CoreNodeModels/RelayCommand.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Windows.Input;
+
+namespace CoreNodeModels
+{
+    public class RelayCommand : ICommand
+    {
+        private readonly Action<object> execute;
+        private readonly Func<object, bool> canExecute;
+
+        public event EventHandler CanExecuteChanged;
+        // {
```

Comment:
what is this?

Reply:
I have removed RelayCommand and created AddMenuItemCommand and RemoveMenuItemCommand classes explicitly. These are nested inside CustomSelectionNodeModel and CustomSelectionItemViewModel classes respectively to limit their usage scope